### DISCUSSION TITLE
Increasing the size of the tabs

### DISCUSSION THREAD

**Discussion by ApurvaSolanki54** [fltk-rs/fltk-rs#717] [Q&A]
I have created 3 tabs on top of my gui window but it only covers a portion of the top of the window and I want to increase the width of the tabs. Any idea what could be done? 
For example, one of your window which has two tabs and i want to increase width so that it covers entire size of top of  the window and the tabs should be of equal sizes. 
![tabs](https://user-images.githubusercontent.com/70524285/121000284-a95f6800-c7a7-11eb-99a8-e2b2c88d6867.jpg)

**Comment by MoAlyousef** [comment]
I don’t think FLTK support changing the size of the tab button, passing spaces to the group label increases the width of the tab. Another option is to create your own tab widget like in the [closable_tab](https://github.com/fltk-rs/fltk-rs/blob/master/fltk/examples/closable_tab.rs) example.
